Wavefront.SDK.CSharp/Common/Constants.cs

```diff
@@ -80,6 +80,11 @@ public static class Constants
         /// </summary>
         public const string ComponentTagKey = "component";
 
+        /// <summary>
+        /// Tag key for defining a process.
+        /// </summary>
+        public const string ProcessTagKey = "proc";
```

Comment:
maybe we should just call it pid instead of proc

Reply:
updated to be pid